testv2/testv2_management/test_management_client.py

```
@@ -706,110 +722,582 @@ def test_token_manager_error_handling(self, mock_token_manager_class, mock_api_c
+                assert len(w) == 1
+                assert issubclass(w[0].category, DeprecationWarning)
+                assert "create_role()" in str(w[0].message)
+                
+            assert result is not None
 
-    def test_api_endpoints_constant(self):
-        """Test that API_ENDPOINTS constant is properly defined."""
-        from kinde_sdk.management.management_client import ManagementClient
+    @patch.object(ManagementTokenManager, 'get_access_token', return_value="fake_token")
+    @patch.object(ManagementTokenManager, '__init__', return_value=None)
+    def test_deprecated_update_role_wrapper(self, mock_tm_init, mock_get_token):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_tm_init`

(ARG002)